app/cdap/testids/testids.yaml

```diff
@@ -0,0 +1,17 @@
+---
+# Please keep this file alphabetically sorted.
+# Keys in this file can contain only alphanumeric characters and underscore.
+
+features:
+  entityListView:
+    justAdded:
+    allEntities:
+    entity:
```

Comment:
This is not in alphabetical order

Reply:
Sorted.

Reply:
Added a custom webpack loader to keep this file sorted. Will add more compile time validations as and when necessary.